src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -555,7 +555,7 @@ class RsFunctionStub(
             flags = BitUtil.set(flags, CONST_MASK, psi.isConst)
             flags = BitUtil.set(flags, UNSAFE_MASK, psi.isUnsafe)
             flags = BitUtil.set(flags, EXTERN_MASK, psi.isExtern)
-            flags = BitUtil.set(flags, VARIADIC_MASK, psi.isExtern)
+            flags = BitUtil.set(flags, VARIADIC_MASK, psi.isVariadic)
```

Comment:
Good catch! 👍 

But we should increase stub version (see `org.rust.lang.core.stubs.RsFileStub.Type#getStubVersion`) because of this and changes in `isUnsafe` function to recalculate stubs and avoid potential incorrect annotations using old stubs